apps/condo/domains/organization/schema/plugins/addOrganizationFieldPlugin.js

```diff
@@ -29,6 +29,11 @@ const addOrganizationFieldPlugin = ({ fromField, isRequired }) => plugin(({ fiel
                     resolvedData['organization'] = get(objWithOrganization, 'organization')
                 }
 
+                // If we cannot resolve organization from parent object -> set organization to null if it is possible
+                if (objWithOrganizationId === null && !isRequired) {
```

Comment:
probably it's ok not to check if this field is required or not ...

Reply:
No, since if the field is required -> we can not set it to NULL